addons/website_forum/views/website_profile_templates.xml

```diff
@@ -100,9 +111,18 @@
                                 <t t-call="website_forum.display_post_answer"/>
                             </div>
                         </t>
-                        <t t-else="">
-                            <span class="fw-bold">No answer posted yet.</span>
-                        </t>
+                        <div t-elif="my_profile" class="text-muted">
+                            You have not answered any questions yet. <br />
+                            <a href="/forum/" class="btn btn-link px-0">
+                                <i class="oi oi-arrow-right d-inline-block"/> Go to Forums
+                            </a>
+                        </div>
+                        <div t-else="" class="text-muted">
+                            This user hasn't answered any questions yet. <br/>
```

Comment:
I bet this "s" is useless (haha, ça rime)

Reply:
How much?